doozer/doozerlib/source_modifications.py

```diff
@@ -125,7 +125,7 @@ def act(self, *args, **kwargs):
                 else:
                     raise ValueError("Unknown 'validate' value: {self.validate}")
         else:  # use doozer local source
-            content = Path(Path(__file__).parent, self.doozer_source).read_text()
+            content = Path(Path(__file__).parent, self.doozer_source).read_text().encode()
```

Comment:
Instead of calling `read_text` then `encode`, we need to use `read_bytes` here in case it is a binary file. Otherwise LGTM

Reply:
updated